### PR TITLE
Disable emoji reacts & prepare 1.9.2

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/components/conversation/message/message-item/GenericReadableMessage.tsx [hunk 1]
```diff
@@ -186,7 +188,7 @@ export const GenericReadableMessage = (props: Props) => {
   );
 
   useEffect(() => {
-    if (msgProps?.convoId) {
+    if (window.sessionFeatureFlags.useEmojiReacts && msgProps?.convoId) {
```
Comment (yougotwill) [inline ts/components/conversation/message/message-item/GenericReadableMessage.tsx:191]: Shouldn't we use enableReactions here?

File: ts/receiver/dataMessage.ts [hunk 2]
```diff
@@ -322,21 +322,22 @@ async function handleSwarmMessage(
...
-        isOpenGroup: false,
-      });
-      if (
-        convoToAddMessageTo.isPrivate() &&
-        msgModel.get('unread') &&
-        rawDataMessage.reaction.action === Action.REACT
-      ) {
-        msgModel.set('reaction', rawDataMessage.reaction as Reaction);
-        convoToAddMessageTo.throttledNotify(msgModel);
+      if (window.sessionFeatureFlags.useEmojiReacts) {
```
Comment (yougotwill) [inline ts/receiver/dataMessage.ts:325]: We should add window.sessionFeatureFlags.useEmojiReacts to the `hasReactions` function in the conversation model and then use it here.

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/components/conversation/message/message-item/GenericReadableMessage.tsx (hunk 1 above)
Comment (Bilb) [reply ts/components/conversation/message/message-item/GenericReadableMessage.tsx:191]: :+1:

File: ts/receiver/dataMessage.ts (hunk 2 above)
Comment (Bilb) [reply ts/receiver/dataMessage.ts:325]: :+1:

**Pull request closed without merging**